doc/manual.rst

```diff
@@ -6020,6 +6020,27 @@ In the example a new pragma named ``rtl`` is introduced that either imports
 a symbol from a dynamic library or exports the symbol for dynamic library
 generation.
 
+noRewrite pragma
+----------------
+
+Term rewriting macros and templates are currently greedy and
+they will rewrite as long as there is a match.
+So there was no way to ensure some rewrite happens only once,
+eg. when rewriting term to same term plus extra content.
+
+noRewrite pragma can actually prevent further rewriting on marked code,
+eg. with given example ``echo("ab")`` will be rewritten just once:
+
+Example:
```

Comment:
No need for `Example:` since the previous line begins with `e.g.` and finishes with a colon.